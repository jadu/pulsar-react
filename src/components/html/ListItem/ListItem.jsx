import React from 'react';

export default class ListItem extends React.Component {

  render() {
    let {
      children,
      ...props
    } = this.props;

    return (
      <li {...props}>
        {children}
      </li>
    );
  }
}