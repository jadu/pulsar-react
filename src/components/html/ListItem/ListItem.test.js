'use strict';

import React from 'react';
import { render } from 'enzyme';

import ListItem from './ListItem';

it('renders the base elements', () => {
  const tree = render(
    <ListItem>foo</ListItem>
  );
  expect(tree).toMatchSnapshot();
});