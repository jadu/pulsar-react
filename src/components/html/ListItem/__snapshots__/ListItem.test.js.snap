// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the base elements 1`] = `
<li>
  foo
</li>
`;
